Source/JavaScriptCore/wasm/WasmFunctionParser.h

```diff
@@ -2451,10 +2525,10 @@ FOR_EACH_WASM_MEMORY_STORE_OP(CREATE_CASE)
 
         ExtSIMDOpType op = static_cast<ExtSIMDOpType>(simdOp);
```

Comment:
Please declare:
```
static constexpr bool isReachable = true;
```
here, and use `isReachable` below instead of the literal `true`.